CrewChiefEx/CrewChiefEx.py

```diff
@@ -128,23 +138,46 @@ def acMain(ac_version):
 
 def acUpdate(deltaT):
     global siminfo
-    global l_lapcount, lapcount,l_driver,l_drivers,maxSlotId,l_flag
-    updateSharedMemory()
+    global l_lapcount, lapcount,l_driver,l_drivers,maxSlotId,l_flag, timer1
+    
+    timer1 += deltaT
+    
+    if timer1 > 0.050:
```

Comment:
should all the logic below this line be in the 'if' block? Perhaps i'm reading it wrong but it looks like you only call updateSharedMemory() if time1 > 0.05, but you call siminfo.getSharedmem() and all the other stuff (including splineToDistanceRoundTrack) even if timer1 <=0.05. Do we need to do all this other stuff ever time AC calls acUpdate(deltaT), or should we only do it if timer1 > 0.05?